.storybook/addons/StoryTheme/StoryTheme.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ColorScheme, { schemes } from '../../../src/components/ColorScheme/ColorScheme';
+
+/*
+	A decorator component that provides baseline conditions required to
+	display library components. All story components are automatically
+	wrapped (see .storybook/preview.js)
+
+	https://storybook.js.org/docs/addons/introduction/#1-decorators
```

Comment:
This just lands on the default addons page and there's no mention of decorators. Maybe this instead? 
```suggestion
	https://storybook.js.org/docs/react/writing-stories/decorators
```

Reply:
ah yeah, this was bad copy/paste anyway